第四次实验/task3.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
In task3.sh line 26:
        top=$(awk -F '\t' '{ if($6=="'$i'") {count[$5]++}} END { for(c in count) {print count[c],c}}' "$1"|sort -nr -k1 | head -n 10)
                                      ^-- SC2086: Double quote to prevent globbing and word splitting.


In task3.sh line 32:
    top=$(awk -F '\t' '{if($5=="'$2'") {count[$1]++}} END {for(c in count) {print count[c],c}}' "$1" |sort -nr -k1 |head -n 10)
                                 ^-- SC2086: Double quote to prevent globbing and word splitting.


In task3.sh line 37:
    echo "Usage: bash "$0" [options]"
                       ^-- SC2027: The surrounding quotes actually unquote this. Remove or escape them.
                       ^-- SC2086: Double quote to prevent globbing and word splitting.